react-lessonform/src/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import ReactDOM, { render } from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.css';
+import './index.css';
+
+class Course extends React.Component {
```

Comment:
```suggestion
class Course extends Component {
```

Reply:
You're already importing it like this `import React, { Component } from 'react';` so you dont need to say  `React.Component`